app/src/analysis/queries/RawMessageQuery.ts

```diff
@@ -20,7 +20,7 @@ const getCoreQuery = () => {
             text AS message,
             is_from_me,
             human_readable_date,
-            contact_name,
+            COALESCE(contact_name, id) as contact_name,
```

Comment:
@Teddarific fyi i think you always wanna coalesce so you get the name OR the phone number